build-logic-commons/basics/src/main/kotlin/gradlebuild.minify.gradle.kts

```diff
@@ -23,9 +23,12 @@ import gradlebuild.basics.transforms.Minify
  * classes and the classes they depend on. The classes are not relocated, they all
  * remain in their original namespace. This reduces the final Gradle distribution
  * size and makes us more conscious of which parts of a library we really need.
+ *
+ * WARNING: these changes might break things in subtle ways.
+ * Existing tests might not cover all use cases.
```

Comment:
This comment doesn't add much value. What shall we do to verify the change if current tests are not enough? Build some local distribution and run some local builds?